lib/plugins/fileStorage/AwsS3Storage.js

```diff
@@ -36,12 +36,16 @@ class AwsS3Storage {
     request.send(function(error, data) {
       // We must alert the write stream that we have finished or it will
       // not close the connection to the client.
-      writeStream.emit('close');
       if (error) {
-        self.logger.error(error);
+        self.logger.error(error, '💀');
```

Comment:
Logged errors should be descriptive and have whatever context we have available. I just put in the emoji for quick debugging (helpful to spot at a glance).